rpxdock/app/dock.py

```diff
@@ -70,7 +77,7 @@ def dock_onecomp(hscore, **kw):
    if kw.docking_method == 'grid':
       flip=list(spec.flip_axis[:3])
       if not kw.flip_components[0]:
-         flip = None
+         flip = None   
```

Comment:
kw.flip_components defaults to True when running rpxdock through dock.py, it will never be an empty list or not defined, so if not kw.flip_components[0] will always be False. I'm not sure what dock does with flip here but we should probably re-evaluate this logic. Either in this PR or in another one.